### DISCUSSION TITLE
jit issue - CssSyntaxError: <css input>: The `bottom-[-6px]` class does not exist.

### DISCUSSION THREAD

**Discussion by knynkwl** [tailwindlabs/tailwindcss#5057] [Help] [answered]
Looks like when `jit` is **not** enabled, Arbitrary value support doesn't work. When `jit` is enabled it works perfectly.

**Comment by knynkwl** [comment]
How should we build for production if this is the case?

**Comment by simonswiss** [comment, marked as answer]
Hey!

That's exactly how it's intended to work. Arbitrary values like `bottom-[-6px]` will only work with `JIT` mode enabled.

**You don't need to turn JIT off for production.**

Tailwind is built at build/compile-time, so by the time you're shipping your project to production, the CSS is already generated. There is no JavaScript runtime or anything.

In "traditional" Tailwind, one would run `purge` for production. With `JIT`, `purge` becomes part of the dev workflow, and there is no need to remove anything before going to production 👍

If you're generating Tailwind JIT with the Tailwind CLI, you can use the `-m` flag to minify your CSS for production. Most tools that support PostCSS will take care of that for you.

Hope it helps!
